node/src/accountant/db_access_objects/sent_payable_dao.rs

```diff
@@ -268,6 +270,61 @@ impl SentPayableDao for SentPayableDaoReal<'_> {
         Ok(())
     }
 
+    fn replace_record(&self, tx: &Tx) -> Result<(), SentPayableDaoError> {
```

Comment:
Okay, I've done some thinking regarding the concern of yours you told me about. My conclusion is, though, that it shouldn't cause any troubles even if we detect more than one falsely-failed transaction that we'd like to revive. I think we should allow an update possibly catering multiple replacements in a single operation. :pray: I'm taking the responsibility on myself :smiling_face_with_tear: